### DISCUSSION TITLE
Thank you for your work

### DISCUSSION THREAD

**Discussion by ghost** [PIut02/ROG-Zephyrus-G14-GA401IV-Hackintosh#11] [General]
I am just writing to thank you for the work you have done, thanks to you I have managed to get MacOS running on my G14 2020 perfectly. People like you make this computer a great community.

Keep up the good work and keep updating the GITHUB, thank you very much for everything.

**Comment by PIut02** [comment]
Thank you for your support as well. My abilities are limited as an individual, and I hope more people can join the team to maintain this repository.
